docs/testkube-automation.md

```diff
@@ -32,7 +32,7 @@ Instead of Helm you can run any other k8s-native command. In our case: `kubectl
 To install on Linux or MacOs run
```

Comment:
perhaps this should change also?

Reply:
and I think the final step in the workflow should be named something like "Run Tests" ?

Reply:
Done. Have a look at.